tests/src/system/basic/WskSdkTests.scala

```diff
@@ -94,7 +94,7 @@ class WskSdkTests
         val scriptfilename = "wsk_cli_bash_completion.sh"
         var scriptfile = new File(dir.getPath(), scriptfilename)
         try {
-            val stdout = wsk.cli(Seq("sdk", "install", "bashauto"), workingDir = dir, expectedExitCode = SUCCESS_EXIT).stdout
+            val stdout = wsk.cli(wskprops.overrides ++ Seq("sdk", "install", "bashauto"), workingDir = dir, expectedExitCode = SUCCESS_EXIT).stdout
```

Comment:
I understand the need for override on the others but this one is unnecessary - since this command does not need the API host. Any way to tolerate it in this case without being disruptive to the rest?